users/tests.py

```diff
@@ -98,13 +98,13 @@ def owner_profile_as_user():
     return Apartment.objects.first().owner
 
 
-# @pytest.mark.django_db
-# def test_is_seeker(seeker_profile_as_user, owner_profile_as_user):
-#     assert seeker_profile_as_user.is_seeker is True
-#     assert owner_profile_as_user.is_seeker is False
+@pytest.mark.django_db
+def test_is_seeker(seeker_profile_as_user, owner_profile_as_user):
```

Comment:
I'm not following, what does this test check?

Reply:
These are not new tests, they relied on the database having the test data and because of that I commented them out when I removed the test data. They were originally added by @tamirho I think when he added the 2 functions in User (is_owner and is_seeker)

Reply:
I still don't understand their purpose. if there is no need for them - they shouldn't be commented on or un-commented. if there's a need for them, you need to be able to explain what is that need and what does this test.

Reply:
They were commented out because they rely on having data in the data-base. and they would have failed because I took out all migration files including the test_data files, now when im adding them back in I added the tests as well

Reply:
Their purpose is to check if the functions `is_seeker` and `is_owner` in User work

Reply:
There are two properties in the `User` model that check whether a user is an owner/seeker
and those are the tests for that functionality.
Please refer to #84 PR.

Reply:
I won't hold back this PR because of it as It's not 100% related and long overdue, but please take care of this comment in a different PR: 
The fixture that is used in this test (both `owner_profile_as_user` and `seeker_profile_as_user`) assumes that there are already owner and seeker users in the DB. tests shouldn't assume such things, and instead, create/imitate the data it is expected to have.